receiver/statsdreceiver/receiver.go

```diff
@@ -46,16 +46,21 @@ func newReceiver(
 	config Config,
 	nextConsumer consumer.Metrics,
 ) (receiver.Metrics, error) {
+	trans := transport.NewTransport(strings.ToLower(string(config.NetAddr.Transport)))
+
 	if config.NetAddr.Endpoint == "" {
-		config.NetAddr.Endpoint = "localhost:8125"
+		if trans == transport.UDS {
+			config.NetAddr.Endpoint = "/var/run/statsd-receiver.sock"
```

Comment:
Will this default be documented in the receiver's docs for the users?

Reply:
Thanks, added some documentation to the README